### PR TITLE
Auto margins

### PR DESCRIPTION
**Pull request by timkelty**
@simonsmith how does this look to you?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,7 +115,7 @@ breakpoints can be configured using `@custom-media`. For example:
 
 ## Usage
 
-### Centring an element in its container
+### Centering an element in its container
```
Comment (simonsmith) [inline README.md]: UK vs US :)

**Comment by simonsmith** [comment]
This looks good man, nice one

File: README.md (hunk 1 above)
Comment (timkelty) [inline README.md]: Ha! Whoa I didn't even know about that one!

**Comment by simonsmith** [comment]
@timkelty Have you been using this package? Think it could be 1.0?

**Comment by timkelty** [comment]
Looks good in IE 10, 11, Safari, FF, Chrome.
1.0 sounds good to me.

**Comment by simonsmith** [comment]
lol at your reply editing my last comment.

Nice one with the browser tests, feel free to merge and release!

**Pull request merged**